api/permissions.py

```diff
@@ -13,6 +13,11 @@ def has_permission(self, request, view):
         profile = Profile.objects.get(request.user)
         return profile.has_premium()
 
+class HasPhone(permissions.BasePermission):
+    def has_permission(self, request, view):
+        profile = Profile.objects.get(request.user)
+        return profile.has_phone()
+
```

Comment:
question: do we need this separate from `HasPhoneService` permission below?

Reply:
Removed in e4f493fe9123f844f733797911f1c01547b82ff2